pkg/console/starter/starter.go

```diff
@@ -96,7 +84,7 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		kubeClient,
 		resync,
 		informers.WithNamespace(api.TargetNamespace),
-		informers.WithTweakListOptions(tweakListOptions),
+		informers.WithTweakListOptions(tweakListOptionsNOOP),
```

Comment:
Do we even need to call this if we're not setting an options?

Reply:
We can probably just eliminate it at this point.